src/c/types/sequence.c

```diff
@@ -212,6 +236,24 @@ bool serialize_endian_sequence_double(MicroBuffer* buffer, const Endianness endi
     return serialize_endian_array_double(buffer, endianness, array, size);
 }
 
+bool deserialize_endian_sequence_char(MicroBuffer* buffer, const Endianness endianness, char* array, uint32_t* size)
+{
+    deserialize_endian_uint32_t(buffer, endianness, size);
+    return deserialize_array_char(buffer, array, *size);
+}
+
+bool deserialize_endian_sequence_bool(MicroBuffer* buffer, const Endianness endianness, bool* array, uint32_t* size)
+{
+    deserialize_endian_uint32_t(buffer, endianness, size);
+    return deserialize_array_bool(buffer, array, *size);
+}
+
+bool deserialize_endian_sequence_uint8_t(MicroBuffer* buffer, const Endianness endianness, uint8_t* array, uint32_t* size)
+{
+    deserialize_endian_uint32_t(buffer, endianness, size);
+    return deserialize_array_uint8_t(buffer, array, *size);
+}
+
 bool deserialize_endian_sequence_uint16_t(MicroBuffer* buffer, const Endianness endianness, uint16_t* array, uint32_t* size)
 {
     deserialize_endian_uint32_t(buffer, endianness, size);
```

Comment:
This function is not safe. First call can fail but second not. Example: Array of one element when buffer is only 3 bytes left.

Reply:
Done!